azkaban/__main__.py

```diff
@@ -85,13 +88,16 @@
                                 will be run only once.
   -t TIME --time=TIME           Time when a schedule should be run. Must be of
                                 the format `hh,mm,(AM|PM),(PDT|UTC|..)`.
+  -x CRON --cron_exp=CRON       Cron expression to use e.g. "0 30 5 ? * *"
   -u URL --url=URL              Azkaban endpoint (with protocol, and optionally
                                 a username): '[user@]protocol:endpoint'. E.g.
                                 'http://azkaban.server'. The username defaults
                                 to the current user, as determined by `whoami`.
                                 If you often use the same url, consider using
                                 the `--alias` option instead.
   -v --version                  Show version and exit.
+  -z ZONE --timezone=ZONE       Timezone to use (PDT|UTC|..)
```

Comment:
Can you confirm that `PDT` works? The [zone ID reference page](https://docs.oracle.com/javase/8/docs/api/java/time/ZoneId.html) you linked below doesn't include it.

Reply:
Unrecognized timezones get overwritten to default server timezone, including PDT. I've updated the doc.

Reply:
It's not ideal that invalid timezones get silently overwritten - are you aware of a way to check for this?

Reply:
I agree that this is not ideal, but the fix will have to be in Azkaban. I'll make a ticket internally to provide a fix to the Azkaban service to fix this. Since this is Python, I do not see a reasonable way to do a client-side fix.

Reply:
Sounds good, thanks.